components/AppBar/src/ColorThemer/MDCAppBarColorThemer.h

```diff
@@ -22,6 +22,32 @@
  */
 @interface MDCAppBarColorThemer : NSObject
 
+/**
+ Applies a color scheme's properties to an MDCAppBarViewController instance using the primary
+ mapping.
+
+ Uses the primary color as the most important color for the component.
+
+ @param colorScheme The color scheme to apply to the component instance.
+ @param appBarViewController A component instance to which the color scheme should be applied.
+ */
++ (void)applyColorScheme:(nonnull id<MDCColorScheming>)colorScheme
```

Comment:
Not sure why this isn't "Soon to be deprecated" anymore?

Reply:
This is the new API.